### PR TITLE
Fixes #428: Support ci.centos.org build and run in Devtools OSD

### LINKED ISSUE #428: Run Documentation in DevTools OSD
This is not a blocker to public launch.openshift.io, but the repository needs some adjustments to be able to build in ci.centos.org and to be published in the Devtools OSD

We need the following files to exist in this repository (it can be copied/based on what's available in https://github.com/openshiftio/launchpad-frontend):

- `cico_build_deploy.sh`
- `Dockerfile.build`
- `Dockerfile.deploy`
- `openshift/template.yaml`

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Comment by gastaldi-osio** [comment]
@vpavlin Check this out. I tested locally and it runs fine. What else do we need to do to have it built and available in the DevTools OSD?

**Comment by gastaldi-osio** [comment]
export CICO_LOCAL=true

**Comment by gastaldi** [comment]
Fixes https://github.com/openshiftio/appdev-documentation/issues/428

File: cico_build_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+docker exec ${BUILDER_CONT} sh scripts/buildGuides.sh
+
+#BUILD DEPLOY IMAGE
+docker build -t ${DEPLOY_IMAGE} -f Dockerfile.deploy .
+
+#PUSH
+if [ -z $CICO_LOCAL ]; then
+    TAG=$(echo $GIT_COMMIT | cut -c1-${TAG_LENGTH})
+    tag_push "${REGISTRY_URL}:${TAG}"
```
Comment (vpavlin) [inline cico_build_deploy.sh]: Please also add username and password as shown here

https://github.com/openshiftio/launchpad-frontend/pull/160/files#diff-5bf05017b1ea493887ae831ae52f78ebR69

File: cico_build_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    fi
+    docker push ${TARGET_IMAGE}
+
+}
+
+# Exit on error
+set -e
+
+if [ -z $CICO_LOCAL ]; then
+    [ -f jenkins-env ] && cat jenkins-env | grep -e PASS -e GIT > inherit-env
```
Comment (vpavlin) [inline cico_build_deploy.sh]: please add `-e DEVSHIFT` so that we get username and password from credentials sourced

**Comment by gastaldi** [comment]
OK to merge

**Comment by rhoads-zach** [comment]
merged: db2e9f5fd8c749fcfe048bdd5dcfbabae2cc8c7a

**Pull request closed without merging**